src/m365/spo/commands/page/page-get.ts

```diff
@@ -83,6 +83,7 @@ class SpoPageGetCommand extends SpoCommand {
       .then((res: { CanvasContent1: string } | void) => {
         if (res && res.CanvasContent1) {
           const canvasData: any[] = JSON.parse(res.CanvasContent1);
+          pageItemData.CanvasContentJson = res.CanvasContent1;
```

Comment:
Since we're no longer using the HTML parsing, we can remove `CanvasContent1` from the call on line 43.